app/app.go

```diff
@@ -782,6 +782,15 @@ func (app *KitoolsApp) Name() string { return app.BaseApp.Name() }
 
 // BeginBlocker application updates every begin block
 func (app *KitoolsApp) BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock) abci.ResponseBeginBlock {
+
+	if ctx.BlockHeight() <= 13519356 {
```

Comment:
If someone is syncing the chain from 0, will this run until syncing past 13519356?

Reply:
Yes, you are right, I will restrict to blocks 13519355 and 13519356 (even if communityTax was 4% before)